### PR TITLE
Update Introduction document

### PR DESCRIPTION
**Pull request by waldyrious**
Changes in this PR:

- Adjust the opening sentence, to use a more natural-sounding English phrasing
- Replace section about SDKs (most of which are not currently under active development) with a new section about our recently released [code samples repository](https://github.com/uphold/rest-api-examples)
- Remove obsolete links in the API Changes section

### CODE REVIEW

File: _introduction.md [hunk 1]
```diff
@@ -1,22 +1,16 @@
 # Introduction
 
-Welcome to the Uphold API, we're glad to have you! Provided here is all the documentation you need to help you create custom and revolutionary new services powered by the Uphold Platform. With your ingenuity, together we can serve the needs of individuals and organizations across the globe and change the financial services ecosystem forever.
+Welcome to the Uphold API — we're glad to have you! Here you can find all the documentation you need to help you create custom and revolutionary new services powered by the Uphold Platform. With your ingenuity, together we can serve the needs of individuals and organizations across the globe and change the financial services ecosystem forever.
```
Comment (hitmanmcc) [inline _introduction.md:3]: Here I would either go with: `Together, with your ingenuity, we...` or `With your ingenuity; together....

Comment (waldyrious) [reply _introduction.md:3]: I agree. To be honest I'd rewrite this entire sentence to be a little less grandiose, but one step at a time ;)

Comment (hitmanmcc) [reply _introduction.md]: Indeed. :)

**Review by hitmanmcc — APPROVED** [review]

**Review by eduardoconceicao — APPROVED** [review]
LGTM

**Pull request merged**